pkg/services/tree/signature.go

```diff
@@ -143,7 +143,7 @@ func verifyMessage(m message) error {
 	return nil
 }
 
-func signMessage(m message, key *ecdsa.PrivateKey) error {
+func SignMessage(m message, key *ecdsa.PrivateKey) error {
```

Comment:
It needs a comment now.

Reply:
thought about that but racked my brains what words should be here

added, check, please